### PR TITLE
Simplify how version number is determined across the board

### PR DESCRIPTION
**Pull request by ketan**

### CODE REVIEW

File: server/webapp/WEB-INF/vm/shared/_header.vm [hunk 1]
```diff
@@ -19,7 +19,7 @@
 <head>
     <meta http-equiv="content-type" content="text/html; charset=UTF-8"/>
 
-    <link href="$req.getContextPath()/$concatenatedCruiseIconFilePath?#include("admin/admin_version.txt.vm")" rel="shortcut icon" />
+    <link href="$req.getContextPath()/$concatenatedCruiseIconFilePath?${currentGoCDVersion.fullVersion()}" rel="shortcut icon" />
```
Comment (jyotisingh) [inline server/webapp/WEB-INF/vm/shared/_header.vm]: The query param seems redundant here, might as well get rid of it.

File: server/webapp/WEB-INF/vm/auth/login.vm [hunk 2]
```diff
@@ -20,7 +20,7 @@
 <head>
   <meta http-equiv="content-type" content="text/html; charset=UTF-8"></meta>
 
-  <link href="$req.getContextPath()/$concatenatedCruiseIconFilePath?#include("admin/admin_version.txt.vm")"
+  <link href="$req.getContextPath()/$concatenatedCruiseIconFilePath?${currentGoCDVersion.fullVersion()}"
```
Comment (jyotisingh) [inline server/webapp/WEB-INF/vm/auth/login.vm]: redundant query param

File: server/webapp/WEB-INF/vm/shared/_header.vm (hunk 1 above)
Comment (ketan) [inline server/webapp/WEB-INF/vm/shared/_header.vm]: Intentional to bust cache :) Rails does this nicely for us by adding a checksum of the resource in the URL.

File: server/webapp/WEB-INF/vm/auth/login.vm (hunk 2 above)
Comment (ketan) [inline server/webapp/WEB-INF/vm/auth/login.vm]: Intentional to bust cache :) Rails does this nicely for us by adding a checksum of the resource in the URL.

**Comment by ketan** [comment]
@jyotisingh — also deleted `server_version.json` that was used from rails.

**Pull request merged**